src/Sylius/Bundle/ApiBundle/Resources/config/routing/product.yml

```diff
@@ -21,23 +21,23 @@ sylius_api_product_create:
             form: sylius_api_product
 
 sylius_api_product_update:
-    path: /{id}
+    path: /{code}
     methods: [POST, PUT, PATCH]
     defaults:
         _controller: sylius.controller.product:updateAction
         _sylius:
             form: sylius_api_product
```

Comment:
I'm currently working with the products api and fell into the problem that `sylius_api_product` has been renamed in https://github.com/Sylius/Sylius/commit/ac09543f0c51dd3e54b1ec9cb8a2586368f0c83c#diff-d41f1e0d680cc8aae555238e267f8db5. I think you'll have the same problem once you're writing `create`/`update` tests 😄 . Maybe that's something you could fix within this PR?